alerts/elasticsearch-gke/README.md

```diff
@@ -0,0 +1,36 @@
+# Elasticsearch Alerts for GKE
+
+## Yellow cluster status alert
+When the cluster has a "yellow" status, at least one primary shard is missing, and data is missing.
+
+## Red cluster status alert
+When the cluster has a "red" status, at least one primary shard is missing, and data is missing.
+
+## High JVM memory heap usage alert
+When the ratio of heap used to max heap becomes high, then application's performance may start to degrade.
+
+### Creating notification Channels and User Labels
+
+Whether these alert policies are being used as standalones or base templates for a deployment strategy like terraform, one thing that should be utilized is notification channels and user labels.
```

Comment:
Let's update this to explain why it should be used: 

"It is strongly recommended to add notification channels and user labels to the alert policies. The notification channel will set the notification destination if the alert policy is triggered. User labels are used for categorization, and can be used to indicate the severity level." 

Could you please make this edit in the readme's below as well?